requirements.txt

```diff
@@ -1 +1,3 @@
 ansible>=2.9.0
+python3-ovirt-engine-sdk4>=4.4.0
+ovirt-imageio-client>=2.0.5
```

Comment:
There is no such module in pypi. Are you still planning to release that? Otherwise it would be good to use the code from #35 as fallback.

Reply:
Do you have it in plan @nirs?

Reply:
Currently we don't publish ovirt-imgeio in pypi. We can do this but it will not work because it depends on qemu-img rpm, so I don't think this is useful.

We don't plan to support a fallback when qemu-img is not available since it means loss of major functionality. The proper way to use imageio is to install the rpm.

If the ansible script  wants to provide minimal functionallity on a system without the required packages you can have a fallback code that does dump upload and download. But you are basically on your own writing such code since we provided a package that does the right thing, and we don't have capacity to support people who do not use our packages.

Reply:
So would it be possible to have current code as a fallback? Will it work with 4.4 or do we need to detect engine version and act accordingly?